djangocms_blog/admin.py

```diff
@@ -86,7 +86,7 @@ class PostAdmin(PlaceholderAdminMixin, FrontendEditableAdminMixin,
     enhance_exclude = ('main_image', 'tags')
     _fieldsets = [
         (None, {
-            'fields': [['title', 'categories', 'publish', 'app_config']]
+            'fields': [['title', 'categories', 'publish', 'app_config'], ['related', ]]
```

Comment:
I would move ``related`` this into a dedicated fieldset. The list can grow quite long and can be inconvenient having it just above the abstract